src/main/java/eu/europeana/metis/sandbox/service/validationworkflow/HarvestValidationStep.java

```diff
@@ -0,0 +1,58 @@
+package eu.europeana.metis.sandbox.service.validationworkflow;
+
+import eu.europeana.metis.sandbox.common.Status;
+import eu.europeana.metis.sandbox.common.Step;
+import eu.europeana.metis.sandbox.domain.Record;
+import eu.europeana.metis.sandbox.domain.RecordInfo;
+import eu.europeana.metis.sandbox.domain.RecordProcessEvent;
+import eu.europeana.metis.sandbox.service.record.RecordLogService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The type Harvest validation step.
+ */
+public class HarvestValidationStep implements ValidationStep {
+    private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
I can give an answer to this :smile: .
This is the way I have started to initialize the loggers because they are much easier to copy/paste and zero error prone.
I have noticed many times a logger being copied from another class that still points to that other class which is incorrect and create wrong logging information.